packages/core/src/components/menu/menu.tsx

```diff
@@ -620,11 +607,15 @@ export class Menu {
               ></ix-menu-expand-icon>
             )}
             {this.breakpoint === 'sm' &&
-              this.applicationLayoutContext.appSwitchConfig && (
+              this.applicationLayoutContext?.appSwitchConfig && (
                 <ix-icon-button
-                  onClick={() =>
-                    showAppSwitch(this.applicationLayoutContext.appSwitchConfig)
-                  }
+                  onClick={() => {
+                    if (this.applicationLayoutContext?.appSwitchConfig) {
```

Comment:
Why do we need this query `if (this.applicationLayoutContext?.appSwitchConfig) {`. Shouldn't this be covered with line 610?